src/mme-app/actionHandlers/defaultMmeProcedureActionHandlers.cpp

```diff
@@ -316,7 +314,8 @@ ActStatus ActionHandlers::default_service_req_handler(ControlBlock& cb)
 	}
 	
 	mmCtxt->setEcmState(ecmConnected_c);
-
+	log_msg(LOG_INFO, "value of enbid %d\n",msgData_p->s1ap_enb_ue_id);
```

Comment:
Lets add new logs more meaningful. With IMSI, and some more details. Just giving enb_ueid may not be of use.

Reply:
Removed this log.